bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -446,7 +438,7 @@ class PlotCanvasView extends Renderer.View
 
     for tool_view in tool_views
       level = tool_view.model.level
-      @levels[level][tool_view.model.id] = tool_view
+      @levels['tool'][tool_view.model.id] = tool_view
```

Comment:
@bigreddot, where this knowledge comes from? Tools shouldn't participate in `render_levels()` at all and this is just a temporary measure before proper implementation emerges. What you may be speaking about are tool introduced (so called, synthetic) annotations, which, as they are renderers, may belong to different levels of rendering (usually `overlay` or `underlay`). Originally those levels were defined on tools, but this was wrong and inconsistent with `bokeh.models`.